app/scripts/modules/core/src/notification/modal/editNotification.less

```diff
@@ -0,0 +1,13 @@
+.slack-channel-form-control {
+  .slack-channel-annotation {
```

Comment:
if you use a <div> since we are anyway positioning it absolute, we dont need the display: block.

Reply:
yeah you're right, i don't even remember why i picked a span in the beginning.